### ISSUE TITLE
Missing information how to install and run it

### ISSUE THREAD

**Issue by misiekofski** [postmanlabs/curl-to-postman#27]
I would like to use it, but don't know what to do. Cloned repo, did npm install and then what?

**Comment by SammyMohamed-FD** [comment]
Readme shows ./curl2postman but no idea where I would run that. Project route contains no such binary and no instructions on how to install

**Comment by louisremi** [comment]
That seems like an interesting tool. I found it through postman's blog. Unfortunately I can only agree: how the hell are we supposed to use it? The readme doesn't help at all.

**Comment by arlemi** [comment]
Agreed that this Readme needs some work, in the meantime you can use Postman directly to convert a curl command to a Postman request, then export it if you want to have it as a JSON file. 

![importcurl](https://user-images.githubusercontent.com/5029719/108838282-592f1200-75cb-11eb-95eb-8f783734c41e.gif)

**Comment by louisremi** [comment]
Thanks @arlemi! I googled ["postman test curl"](https://www.google.com/search?channel=fs&client=ubuntu&q=postman+test+curl) and the first result is [an old blog post from 2016] that points to this repo. It seems you now have a better way tu use cURL right in postman. Maybe you could update that blog post to reflect that?

**Comment by michal-dobrzycki-coderslab** [comment]
>  in the meantime you can use Postman directly to convert a curl command to a Postman request

The problem I was trying to solve was running that import for 2k+ curls (don't ask me how I got there) ;) 
Pasting manually to Postman was (and still is) not an option. I thought this repo would help me

**Comment by arlemi** [comment]
@louisremi That poor misses... a lot of info indeed haha. Good catch, we'll get that updated! 

@michal-dobrzycki-coderslab Indeed, probably don't want to do it one-by-one, I found that package that you can use to convert several curl commands into one collection: https://github.com/bhargavkaranam/multiple-curl-to-postman
- clone the repo
- run `npm install`
- run `node index.js curl1 curl2 curl3...`

**Comment by nimit2801** [comment]
I have a few examples for this repo!
I'll open a PR and list them in README.md

**Comment by MrGitHumbug** [comment]
It was my 1st try at making a api for NOW token/wallet

**Comment by MrGitHumbug** [comment]
If you would like fresh copy's look up NOW token API

**Comment by MrGitHumbug** [comment]
I had no idea what I was doing. Just followed a how to guide, badly written. Sorry to disappoint but it's for NOW token not trust wallet. I just had some stored in my trust wallet. As in the native wallet to ChangeNOW exchange
